_playwright-tests/Integration/CanInstallUploadRepositryOnHost.spec.ts

```diff
@@ -0,0 +1,135 @@
+        await cleanup.runAndAdd(() => cleanupTemplates(client, templateNamePrefix));
+        cleanup.add(() => regClient.Destroy('rhc'));
+      });
+
+      await navigateToTemplates(page);
+      await closePopupsIfExist(page);
+      await expect(page.getByRole('button', { name: 'Create template' })).toBeVisible();
+      await page.getByRole('button', { name: 'Create template' }).click();
+      await page.getByRole('button', { name: 'filter architecture' }).click();
+      await page.getByRole('menuitem', { name: 'x86_64' }).click();
+      await page.getByRole('button', { name: 'filter OS version' }).click();
+      await page.getByRole('menuitem', { name: 'el9' }).click();
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      // skipping red hat repositories step since we are using upload repository
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      await expect(
+        page.getByRole('heading', { name: 'Other repositories', exact: true }),
+      ).toBeVisible();
+      await page.getByRole('searchbox', { name: 'Filter by name' }).fill(uploadRepo);
+      const rowUploadRepo = await getRowByNameOrUrl(page, uploadRepo);
+      await expect(rowUploadRepo.getByText('Valid')).toBeVisible({ timeout: 60000 });
+      await rowUploadRepo.getByLabel('Select row').click();
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      await page.getByText('Use the latest content', { exact: true }).click();
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      await expect(page.getByText('Enter template details')).toBeVisible();
+      await page.getByPlaceholder('Enter name').fill(`${templateName}`);
+      await page.getByPlaceholder('Description').fill('Template test for upload repository');
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      await page.getByRole('button', { name: 'Create other options' }).click();
+      await page.getByText('Create template only', { exact: true }).click();
+      const rowTemplate = await getRowByNameOrUrl(page, `${templateName}`);
+      await expect(rowTemplate.getByText('Valid')).toBeVisible({ timeout: 660000 });
+    });
+
+    await test.step('Register system with template using RHSM client', async () => {
+      await regClient.Boot('rhel9');
+
+      const reg = await regClient.RegisterRHC(
+        process.env.ACTIVATION_KEY_1,
+        process.env.ORG_ID_1,
+        templateName,
+      );
+      if (reg?.exitCode != 0) {
+        console.log('Registration stdout:', reg?.stdout);
+        console.log('Registration stderr:', reg?.stderr);
+      }
+      expect(reg?.exitCode).toBe(0);
+
+      await refreshSubscriptionManager(regClient);
+      await runCmd('Clean cached metadata', ['dnf', 'clean', 'all'], regClient);
+    });
+
+    await test.step('Install from the template and verify the upload repository is installed', async () => {
```

Comment:
i'd also say `verify the upload repository content is installed` here

Reply:
sorry i meant `Install from the template and verify the upload repository content is installed`